### ISSUE TITLE
Cannot Connect React UI to backend

### ISSUE THREAD

**Issue by d-e-v-esh** [fossology/FOSSologyUI#192]
### Description

I am not able to connect the React UI to the backend and log in as `fossy`.  Whenever I log in, I am getting this CORS error:

![Screenshot from 2022-03-24 00-14-59](https://user-images.githubusercontent.com/59534570/159773665-100c8488-7be6-42a8-886e-3a30d6969ee0.png)

> Access to fetch at 'http://localhost/repo/api/v2/info' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

1. I have created the .env file.
2. I added `http://localhost:3000` to `Allowed origins for REST API` in the fossology config.

![Screenshot from 2022-03-24 00-15-30](https://user-images.githubusercontent.com/59534570/159774080-2713da20-2cd3-4bd1-bd9e-2a086f3d048c.png)

Still, I am not able to log in. Am I missing something? Please Help 🙏🙏

**Comment by GMishx** [comment]
Duplicate of #181
